gobblin-runtime/src/main/java/gobblin/runtime/app/ServiceBasedAppLauncher.java

```diff
@@ -103,6 +104,7 @@ public ServiceBasedAppLauncher(Properties properties, String appName) throws Exc
     this.services = new ArrayList<>();
 
     // Add core Services needed for any application
+    addStateStoreCleaner(properties);
```

Comment:
I am going to remove this until state store cleaner works for all types of state stores.